Versions/Common/BeStride_ActionButton.lua

```diff
@@ -8,7 +8,7 @@ function BeStride:CreateActionButton(buttontype)
 	local name = "BeStride_AB" .. buttontype .. "Mount"
 
 	local br = CreateFrame("Button", name, UIParent, "SecureActionButtonTemplate,ActionButtonTemplate")
-	br:RegisterForClicks("AnyUp", "AnyDown")
```

Comment:
The whole addon logic was called twice on each key press, I don't see the point of it, but I might be missing something that has an impact somewhere else.

Reply:
It's undirectly related because otherwise, it breaks the dismount logic I added. As the code is ran twice, it cancels the form on the first call and then reapply the form instantly on the second call.

From what I gathered online, this was to fix a bug currently live on retail (https://us.forums.blizzard.com/en/wow/t/dragonflight-click-bindings-broken/1361972/28). From what I understand, it makes the button clicks only happening on Up event, instead of defaulting to this CVar (https://wowpedia.fandom.com/wiki/CVar_ActionButtonUseKeyDown).

I think to fix it for both Retail and Classic, we could simply add `br:RegisterForClicks("AnyUp", "AnyDown")` for the Mainline version.

Reply:
thanks for this.